### PR TITLE
added common business date acronyms

### PR DESCRIPTION
**Pull request by gravityvi**
### Related Issues
- #251

### CODE REVIEW

**Review by djmitche — COMMENTED** [review]
This looks good!  Can you add some tests?

File: docs/src/time.md [hunk 1]
```diff
@@ -28,3 +28,18 @@ Some of the units allow an adjectival form, such as `daily` or `annually`; this
...
+
+Some commonly used named timestamps
+
+ * `today` Start of today
+ * `yesterday` Start of yesterday
+ * `tomorrow` Start of tomorrow
+ * `sod` Start of today
+ * `eod` End of today
+ * `sow` Start of the next week
+ * `eow` End of the week
```
[the remark this answers is not in the document]
Comment (gravityvi) [reply docs/src/time.md:43]: I have added an image in docs for better understanding

[the remark this answers is not in the document]
Comment (gravityvi) [reply docs/src/time.md:43]: name_timestamp

**Review by djmitche — APPROVED** [review]
I'll merge and we'll address the clippy issue separately -- @dbr's suggestion is a good one.

**Pull request merged**